### DISCUSSION TITLE
Campus Expert Mail Received After Deadline

### DISCUSSION THREAD

**Discussion by DhruvDange** [community/community#34984] [GitHub Education] [answered]
Hey guys!
I applied for the github campus experts program in late August this year and was a bit surprised to not have received any mail with updates. I did send out an email to github education support asking for an update but didnt get any update either.

Today, October 4 2022, exactly a month after i was supposed to receive an email with updates on my application, I received an email from github education saying they have accepted my application and would like to move forward!! 

The mail asks me to upload a video introducing myself and my community, however, the last deadline for the video is October 2 2022. 
Since I received the mail AFTER the deadline, can someone help me out with extending the deadline please.
![unnamed](https://user-images.githubusercontent.com/46223201/193661400-37816b7f-f787-4b79-8a13-7e4d8c04ca68.jpg)

Thank you

**Comment by KenjiPcx** [comment]
Hey @DhruvDange I got the same email as well, genuinely wondering what I should do

**Reply by DhruvDange** [reply]
Same here! If you receive any updates please do let me know too. Thanks :)

**Reply by cosmicqbit** [reply]
Whoa! We are on the same page men @DhruvDange @KenjiPcx and I came here to seek answer. Not sure what to do next. Did anybody among you tried sending a mail back to github about deadline being over already.

**Reply by JavedTadavi** [reply]
Same here

**Comment by cosmicqbit** [comment]
So turns out this was a technical glitch / error. Am sure you also received this email by now.

<img width="788" alt="image" src="https://user-images.githubusercontent.com/75565639/193739552-c18c7cb5-f138-481d-86f1-10633bf184e1.png">

**Reply by samus-aran** [reply]
Thanks @cosmicqbit for updating the community with this! 👍

**Reply by cosmicqbit** [reply]
Can you please look into the mail logs, I know lot of students (myself included) who didn't received any rejection mail or any update from Github Campus at all and they are still waiting for the response not knowing that the first round is already over. Except I received that over due mail which was sent due to technical error in your automated mailing system. Also the URL to check [application status](https://apply.githubcampus.expert/status) either redirects to home URL or displays no updated information like in my case:

<img width="1013" alt="image" src="https://user-images.githubusercontent.com/75565639/194074199-98c2dd36-7963-479b-a0d6-c4d28c3831fb.png">

**Reply by lasr21** [reply]
> Can you please look into the mail logs, I know lot of students (myself included) who didn't received any rejection mail or any update from Github Campus at all and they are still waiting for the response not knowing that the first round is already over. Except I received that over due mail which was sent due to technical error in your automated mailing system. Also the URL to check [application status](https://apply.githubcampus.expert/status) either redirects to home URL or displays no updated information like in my case:
> 
> <img alt="image" width="1013" src="https://user-images.githubusercontent.com/75565639/194074199-98c2dd36-7963-479b-a0d6-c4d28c3831fb.png">

Hi @cosmicqbit please enter again to the page. now you will be able to see your status.

**Reply by cosmicqbit** [reply]
> > Can you please look into the mail logs, I know lot of students (myself included) who didn't received any rejection mail or any update from Github Campus at all and they are still waiting for the response not knowing that the first round is already over. Except I received that over due mail which was sent due to technical error in your automated mailing system. Also the URL to check [application status](https://apply.githubcampus.expert/status) either redirects to home URL or displays no updated information like in my case:
> > <img alt="image" width="1013" src="https://user-images.githubusercontent.com/75565639/194074199-98c2dd36-7963-479b-a0d6-c4d28c3831fb.png">
> 
> Hi @cosmicqbit please enter again to the page. now you will be able to see your status.

The page is now working, hoping it does for all. Thank you for sorting this out anyway :)

**Reply by samus-aran** [reply]
Thanks @lasr21 for the fix 👍 and thanks @cosmicqbit for raising it to our attention 👀 Awesome work.

**Reply by MahadMuhammad** [reply]
Hey, @lasr21 and @samus-aran. I applied to the GitHub Campus Experts program in February 2024. I received an email on Tuesday, March 12, stating that the status would be shared on March 19, 2024. However, I haven't received any email regarding the selection or rejection of my application. Could you kindly advise me on the next steps to take?

**Reply by JavedTadavi** [reply]
Same here bro , If you receive any updates please do let me know

**Comment by samus-aran** [comment, marked as answer]
Hi everyone! 👋

Would like to start off with thanking you all for your understanding and we appreciate you reaching out to us on receiving the above Global Campus Experts email(s).

We want to confirm that there had been an error made on our part and some applicants for Campus Experts had been emailed incorrectly.

When processing the Campus Experts applications we triggered an automation that sent a previously mailed (August) application status update to the applicants. This email detailed the applicant’s status and if applicable the next steps in the application with a deadline to achieve those steps.

Although we identified the issue and quickly canceled the emailing queue we are aware that some of the applicants had received the email in error.

We would recommend that applicants check their [current status](https://apply.githubcampus.expert/status) to get the most up to date information on their application.

We appreciate all the time and effort our applicants have taken in their application and understand that for some this news might not be what they were hoping for. We sincerely apologise for any inconvenience this may have caused.
